app/scripts/components/analysis/results/chart-card.tsx

```diff
@@ -109,19 +124,45 @@ export default function ChartCard(props: ChartCardProps) {
         </CardHeadline>
         <CardActions>
           <Toolbar size='small'>
-            <Tip
-              content={noDownloadReason}
-              disabled={!noDownloadReason}
-              hideOnClick={false}
+            <Dropdown
+              alignment='right'
+              triggerElement={(props) => (
+                <Tip
+                  content={noDownloadReason}
+                  disabled={!noDownloadReason}
+                  hideOnClick={false}
+                >
+                  <ChartDownloadButton
+                    {...props}
+                    variation='base-text'
+                    visuallyDisabled={!!noDownloadReason}
+                  >
+                    <CollecticonDownload2 title='Download' meaningful />
+                  </ChartDownloadButton>
+                </Tip>
+              )}
             >
-              <ChartDownloadButton
-                variation='base-text'
-                onClick={onExportClick}
-                visuallyDisabled={!!noDownloadReason}
-              >
-                <CollecticonDownload2 title='Download' meaningful />
-              </ChartDownloadButton>
-            </Tip>
+              <DropTitle>Select a file format</DropTitle>
+              <DropMenu>
+                <li>
+                  <DropMenuItem
+                    role='button'
+                    onClick={() => onExportClick('image')}
+                  >
+                    Image (JPG)
+                  </DropMenuItem>
+                </li>
+                <li>
+                  <DropMenuItem
+                    role='button'
+                    onClick={() => onExportClick('text')}
+                  >
+                    Text (CSV)
+                  </DropMenuItem>
```

Comment:
This ☝️

Reply:
Can we discuss that? Why is `DropMenuItem` not a `button` ?

Reply:
@nerik Historically speaking `DropMenuItem` was created to be an item that takes you somewhere (link). The dropdown had a list of links - simple as that. However cases like this, where the click triggers something kept on appearing and we kept on using links.

A button would make more sense, but that requires updating the styles since `DropMenuItem` is not prepared to be a button. If you set it as a button (`<DropMenuItem as='button'>`) you'll see things break. I think that "laziness" and the fact that a `<a href='#'>` is not wrong, took the best of us.

However, better late then never I guess. Let's take this as a good opportunity to update these styles and start having buttons

Reply:
@danielfdsilva Ok, makes sense, thanks for the explanation.